samples/snippets/csharp/VS_Snippets_CLR_System/system.IO.Pipes.NamedPipeClientStream_ImpersonationSample1/cs/NamedPipeClientStream_ImpersonationSample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Solution files are not needed for single projects.

Reply:
I'll remove it

Reply:
Btw, it might worth having a solution grouping this project and the server sample one...

I can create both the csproj and the sln to group these 2 samples, but I might need your guidance to tell me where it would be the best path to store the sln file.

Reply:
@nemrism 

> I can create both the csproj and the sln to group these 2 samples, but I might need your guidance to tell me where it would be the best path to store the sln file.

I would move both samples under the folder with the article, in a *snippets* folder. Then, I'd put the solution in one folder, and each project in a subfolder of that solution folder. 

If you want to take that on, I'll watch for a new PR.

Reply:
I'll do that in a new branch in a few

Reply:
> If you want to take that on, I'll watch for a new PR.

I'll do that in a new branch in a few, thanks @BillWagner